build

```diff
@@ -1272,6 +1287,10 @@ for RECIPEFILE in "${RECIPEFILES[@]}" ; do
     if test -n "$VM_TYPE" -a -z "$RUNNING_IN_VM" ; then
 	vm_first_stage
 	cleanup_and_exit
+    else
```

Comment:
No need for the "else", cleanup_and_exit exits

Reply:
yup, agreed